### PR TITLE
emit `id_token`s when we refresh tokens.

### PR DESCRIPTION
**Pull request by sbalaji1996**
## Changes made

When we make the request to `https://accounts.google.com/o/oauth2/token` to
refresh a user’s tokens, emit the refreshed id token that comes back as well,
such that consumers of this API can keep both those values in sync.

### CODE REVIEW

File: .travis.yml
Comment (campriceaustin) [inline .travis.yml:4]: wow

File: test/xoauth2-test.js
Comment (campriceaustin) [inline test/xoauth2-test.js:127]: nice tests

**Review by campriceaustin — APPROVED** [review]

File: package.json
Comment (wearhere) [inline package.json:28]: I take it that this was using a global install previously?

File: src/xoauth2.js
Comment (wearhere) [inline src/xoauth2.js:104]: good documentation

File: src/xoauth2.js
Comment (wearhere) [inline src/xoauth2.js]:
```suggestion
 * Emits 'token': { user: User email-address, accessToken: the new accessToken, timeout: TTL in seconds, idToken: the new idToken}
```

File: src/xoauth2.js
Comment (wearhere) [inline src/xoauth2.js:173]: Curious do you have any concerns with providing it to the callback? It might prove useful to clients.

File: .travis.yml
Comment (wearhere) [inline .travis.yml:9]: Btw I can understand removing this info from our fork. At the same time, as I mentioned live, I bet Andris would take a PR. We know him personally and could reach out if you liked.

File: package.json
Comment (sbalaji1996) [reply package.json:28]: Presumably, though the travis config doesn't install it in `before_install`.

File: src/xoauth2.js
Comment (sbalaji1996) [reply src/xoauth2.js:173]: I just wanted to limit the scope of my changes given how we use this library w.r.t. sending - also since I'd forked this for our internal usage (where we only need the id token to store later), it struck me as somewhat spurious to make that change.

File: .travis.yml
Comment (sbalaji1996) [reply .travis.yml:9]: I'm open to that, sure - however, I'd like to get this out ASAP given that it fixes a regression. Could we email him and see if we could expedite a review?

Comment (wearhere) [reply .travis.yml:9]: Sure! Brad has his email.

You can also merge this into our fork now and submit a "copy" to the upstream repo, and later reconcile the two.

**Review by shils — APPROVED** [review]
LGTM once the JSDoc for `XOAuth2Generator.prototype.updateToken` has been updated

File: .travis.yml
Comment (sbalaji1996) [reply .travis.yml:9]: Cool - emailed. I'll wait until tonight to see if he responds, otherwise I'll merge and deploy tomorrow morning.

**Pull request merged**
